ecs-agent/tmds/utils/netconfig/netconfig_linux.go

```diff
@@ -24,6 +24,16 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
+type NetworkConfigClient struct {
```

Comment:
non-blocking: this object can be declared locally.

Reply:
Hm I think it's still needed since it's referenced outside of `netconfig` package

Reply:
Ah I see.